docs/content/commands/npm-uninstall.md

```diff
@@ -7,7 +7,7 @@ description: Remove a package
 ### Synopsis
 
 ```bash
-npm uninstall [<@scope>/]<pkg>[@<version>]... [-S|--save|-D|--save-dev|-O|--save-optional|--no-save]
+npm uninstall [<@scope>/]<pkg>[@<version>]... [--no-save]
```

Comment:
`--save` is still important to document, in case someone has `save=false` in their npmrc (as i often do)

Reply:
Ah! Thanks for the clarification. I had overlooked that in my local testing.